tournaments/index.js

```diff
@@ -835,6 +836,9 @@ class Tournament {
 		}
 	}
 	onBattleWin(room, winnerid) {
+		delete room.tour;
+		delete room.parent;
```

Comment:
`delete` should never be used and is also against the room spec. Set them to `null` instead.

Reply:
Should that also be changed in the place that i copied that from?